insights/parsers/software_collections_list.py

```diff
@@ -0,0 +1,67 @@
+"""
+Software Collections list output - command ``scl --list``
+=========================================================================
+
+This module provides parser for list output of ``scl``.
+
+Parser provided by this module is:
+
+SoftwareCollectionsListInstalled - command ``scl --list``
+------------------------------------------------------------------------------------
```

Comment:
Since you already have this information in the title, this sub heading is not really necessary but its not necessary to change it unless you are making other changes.